src/app/store/rubrics.reducer.spec.ts

```diff
@@ -0,0 +1,258 @@
+import { DisplayableErrorInterface } from '../object-interfaces/displayable-error.interface';
+import { RubricInterface } from '../object-interfaces/rubric.interface';
+import {
+    ConnectionLost,
+    ConnectionMade,
+    CreateRubric,
+    CreateRubricError,
+    CreateRubricSuccess,
+    DeleteRubric,
+    DeleteRubricError,
+    DeleteRubricSuccess,
+    GetRubric,
+    GetRubricError,
+    GetRubrics,
+    GetRubricsError,
+    GetRubricsSuccess,
+    GetRubricSuccess,
+    SetPageTitle,
+    UpdateCurrentRubric,
+} from './rubrics.actions';
+import { rubricsReducer } from './rubrics.reducer';
+import { getInitialState } from './rubrics.state';
+
+describe('RubricsReducer', () => {
+    const { rubrics: initialState } = getInitialState();
+    const rubric: RubricInterface = { name: 'test' };
+    const currentRubric: RubricInterface = { maxLevelsCount: 0, name: 'test', totalPoints: 0 };
+    const allRubrics: Array<RubricInterface> = [];
+    const error: DisplayableErrorInterface = { message: 'test' };
```

Comment:
When running multiple tests within a describe block like this, you should use [`beforeEach`](https://jasmine.github.io/2.0/introduction.html#section-Setup_and_Teardown).

You should still make the `const` declarations at the top of the block like this, but by setting the values in the `beforeEach` block, you can guarantee that if any values are changed within the tests, they will be reset.

Does that make sense? If not, feel free to grab me today and we can go over it.